test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetTrustTests.cs

```diff
@@ -0,0 +1,564 @@
+
+            using (SimpleTestPathContext pathContext = _msbuildFixture.CreateSimpleTestPathContext())
+            using (MemoryStream zipStream = await package.CreateAsStreamAsync())
+            using (TrustedTestCert<TestCertificate> trustedTestCert = SigningTestUtility.GenerateTrustedTestCertificate())
+            {
+                string certFingerprint = SignatureTestUtility.GetFingerprint(trustedTestCert.Source.Cert, HashAlgorithmName.SHA256);
+                string signedPackagePath = await SignedArchiveTestUtility.AuthorSignPackageAsync(trustedTestCert.Source.Cert, package, pathContext.PackageSource);
+                var config = $@"<?xml version=""1.0"" encoding=""utf-8""?>
+                    <configuration>
+                      <packageSources>
+                        <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+                        <clear />
+                        <add key=""NuGetSource"" value=""{pathContext.PackageSource}"" />
+                       </packageSources>
+                      <config>
+                        <add key=""signaturevalidationmode"" value=""accept"" />
+                      </config>
+                      <trustedSigners>
+                      </trustedSigners>
+                    </configuration>";
+                SettingsTestUtils.CreateConfigurationFile(nugetConfigFileName, pathContext.WorkingDirectory, config);
+                var nugetConfigPath = Path.Combine(pathContext.WorkingDirectory, nugetConfigFileName);
+                var allowUntrustedRootArg = allowUntrustedRoot ? "--allow-untrusted-root" : string.Empty;
+                var allowUntruestedRootValue = allowUntrustedRoot ? "true" : "false";
+
+                // Act
+                CommandRunnerResult resultAdd = _msbuildFixture.RunDotnet(
+                    pathContext.SolutionRoot,
+                    $"nuget trust author nuget {signedPackagePath}  {allowUntrustedRootArg} --configfile ..{Path.DirectorySeparatorChar}{nugetConfigFileName}");
+
+                // Assert
+                resultAdd.Success.Should().BeTrue();
+                resultAdd.AllOutput.Should().Contain(string.Format(CultureInfo.CurrentCulture, _successfulAddTrustedSigner, "author", "nuget"));
+
+                string expectedResult = SettingsTestUtils.RemoveWhitespace($@"<?xml version=""1.0"" encoding=""utf-8""?>
+                    <configuration>
+                      <packageSources>
+                        <!--To inherit the global NuGet package sources remove the < clear /> line below-->
+                        <clear/>
+                        <add key = ""NuGetSource"" value = ""{pathContext.PackageSource}""/>
+                       </packageSources >
+                      <config>
+                        <add key = ""signaturevalidationmode"" value = ""accept""/>
+                      </config>
+                      < trustedSigners>
```

Comment:
``` suggestion
                      <trustedSigners>
```

Reply:
Can I do it later since it's just test typo? Might take sometime for get another green build.

Reply:
I have another follow up issue https://github.com/NuGet/Home/issues/10788, then I can include it there.